src/daemon/rpmostreed-daemon.cxx

```diff
@@ -815,6 +815,12 @@ rpmostreed_daemon_is_rebooting (RpmostreedDaemon *self)
   return self->rebooting;
 }
 
+GDBusConnection *
+rpmostreed_daemon_connection(void)
+{
+  g_assert (_daemon_instance->connection);
+  return _daemon_instance->connection;
```

Comment:
Could this be causing all the CI "error: GDBus.Error:org.freedesktop.DBus.Error.NameHasNoOwner: Could not activate remote peer." errors?
Manual testing seems to behave as expected, but the `vmcheck` test suite is not happy.